src/components/BotAppearance/BotAppearance.tsx

```diff
@@ -70,13 +84,22 @@ export const BotAppearancePage = () => {
   useEffect(() => {
     if (sources && !isInitialData.current) {
       isInitialData.current = true
+
+      if (sources.accentColour == null) {
+        sources.accentColour = '#ffffff'
+      }
+
       reset({
         botId: id as string,
+        companyLogoAttachmentId: sources.companyLogoAttachmentId!,
+        botAvatarAttachmentId: sources.botAvatarAttachmentId!,
```

Comment:
Please avoid using ﻿! in the codebase as this pattern is hard to read and may introduce runtime errors. Instead, consider using ﻿if-else statements, ﻿||, ﻿??, or optional chaining even if you are confident that the values always exist.